src/View/View.js

```diff
@@ -0,0 +1,27 @@
+import { Console } from "@woowacourse/mission-utils";
+import Validation from "./Validation.js";
+
+class View {
+  static async readCarsName() {
+    const input = await Console.readLineAsync(
+      "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+    );
+    const carNames = input.split(",").map((name) => name.trim());
+    Validation.validateCarNames(carNames);
+    return carNames;
```

Comment:
```suggestion
    const carNames = input.split(",").map((name) => name.trim());

    Validation.validateCarNames(carNames);
    
    return carNames;
```
1주차 공통 피드백에 나와있는 것처럼 Validation을 수행하는 부분은 사용자 입력을 받는 것과는 문맥이 달라지니 공백을 넣어 구분해주는게 가독성이 좋을 것 같습니다!

Reply:
진짜 꼼꼼하게 봐주셨네요!!! 갬덩
앞으로 공백도 의미있게 써야겠습니당